SpatialGDK/Source/SpatialGDK/Public/Utils/InterestFactory.h

```diff
@@ -70,6 +70,7 @@ class SPATIALGDK_API InterestFactory
 	void AddServerGameplayDebuggerCategoryReplicatorActorInterest(Interest& OutInterest,
 																  const AGameplayDebuggerCategoryReplicator& Replicator) const;
 #endif
+public:
```

Comment:
Is popping open the Interest factory's private API necessary ?

Reply:
Yeah there's no need to do that - I'll restrict it.
The context is @m-samiec's suggestion to restrict skeleton entities' interest to ServerSelfInterest.